confidant/routes/jwks.py

```diff
@@ -35,27 +35,36 @@ def get_token(id):
        Content-Type: application/json
 
        {
-         "token": "ey..."
+         "is_service": true,
+         "requester": "pyexample2",
+         "user": "ey..."
        }
```

Comment:
these are the contents within the JWT payload, but we're still responding as 
```
"token": "ey..."
```

Reply:
thanks.  corrected